bluesky/run_engine.py

```diff
@@ -876,6 +891,13 @@ def _close_run(self, msg):
 
     @asyncio.coroutine
     def _create(self, msg):
+        """Trigger the run engine to start collecting data for an event
+
+        Expected message object is:
+
+            Msg('create', None, name='primary')
+            Msg('create')
```

Comment:
Exactly. The reason this was introduced is explained [here](https://github.com/NSLS-II/bluesky/pull/313#issuecomment-177662029).